src/sentry/api/endpoints/organization_events_anomalies.py

```diff
@@ -68,7 +68,9 @@ def post(self, request: Request, organization: Organization) -> Response:
         """
         Return a list of anomalies for a time series of historical event data.
         """
-        if not features.has("organizations:anomaly-detection-alerts", organization):
+        if not features.has(
+            "organizations:anomaly-detection-alerts", organization
+        ) and not features.has("organizations:anomaly-detection-rollout", organization):
```

Comment:
I thought about making a helper function to do this but ultimately decided against it because it'd be more work to remove when we get rid of the non-rollout flag

Reply:
yep - agree, i think it's fine to leave this kind of stuff around cause at some point we'll want to go through and fully enable everything.